### ISSUE TITLE
[Bug]: Can't complie Swift application on iOS

### ISSUE THREAD

**Issue by IlannM** [theos/theos#792]
### What are the steps to reproduce this issue?

1. Jailbreak an iOS 15.8.1 arm64 device with Dopamine 2.0.5
2. Install `theos-dependencies` package
3. Run the official theos installation script
4. Run NIC and create a `iphone/application_swift` project
5. Edit (fix) the `Makefile`: specify the latest SDK and add  `ARCHS = arm64`
6. run `make`

### What happens?

Make fails to compile, displaying errors such as not finding `/bin/sh`, `Use of uninitialized value $[some variable]`. See indexed screenshot below.

### What were you expecting to happen?

Successfully compile a Swift app on iOS with theos.

### Paste any relevant logs, error output, etc.

```shell
![Screenshot from 2024-02-20 15-20-51](https://github.com/theos/theos/assets/56805451/d6482b5d-017f-44a1-9fd9-f9f0fb6c7029)
```

### What OS are you seeing the problem on?

Linux/WSL

### What OS version does it have installed?

Ubuntu 23.04

### What toolchain and version are you using?

Latest Procursus packages as of Feb 20.

### Which SDK version are you using?

iOS 16.5

### What OS is your client device running?

iOS

### What OS version does it have installed?

15.8.1

### Which device model is it?

iPhone 7 Plus

### Which jailbreak is it using (if relevant)?

Dopamine 2.0.5

**Comment by L1ghtmann** [comment]
Hey, thanks for reporting this. Unfortunately, this appears to be out of our hands as it looks to be an issue with the rootless `perl` having an [incomplete patch](https://github.com/ProcursusTeam/Procursus/issues/1427).

**Comment by IlannM** [comment]
Indeed, the exact same issue. Do you mean to say the perl package itself needs updating/patching from the Procursus maintainers?

**Comment by L1ghtmann** [comment]
That's what appears to be the case, yes.
